src/app/(authenticated)/comic-issue/[id]/page.tsx

```diff
@@ -56,15 +59,19 @@ export default async function ComicIssuePage({ params: { id } }: ComicIssuePageP
             </Text>
             <ChevronRightIcon className='text-grey-600' />
           </Link>
-          <InfoListActions
-            averageRating={comicIssue.stats?.averageRating}
-            className='flex w-fit [&>*]:min-w-20'
-            comicIssueId={comicIssue.id}
-            favouritesCount={comicIssue.stats?.favouritesCount}
-            isFavourite={comicIssue.myStats?.isFavourite}
-            orientation='horizontal'
-            rating={comicIssue.myStats?.rating}
-          />
+          <div className='flex gap-1 md:gap-2 justify-between md:justify-around'>
```

Comment:
we will ignore this layout for now because there is a new design for comic issue details screen